tests/unit/issues/Issue46.d

```diff
@@ -10,12 +10,14 @@ import Common;
 import dstep.translator.Options;
 
 // Fix 46: Generating code that will not compile.
-unittest
+version (Posix)
```

Comment:
Could have used the `version (Posix):` syntax to avoid the extra indentation. But that's a minor issue.